packages/core/src/external/aws/app-config.ts

```diff
@@ -95,3 +100,112 @@ export async function getFeatureFlagValueBoolean<T extends keyof BooleanFeatureF
   const configContentValue = await getFeatureFlags(region, appId, configId, envName);
   return configContentValue[featureFlagName];
 }
+
+export async function getDeployments({
```

Comment:
Based on my comments regarding the latest deployments we may not need this

Reply:
Done.